RULES.md

```diff
@@ -156,6 +157,9 @@ Rules from the AWS NIST 800-53 Conformance Pack which could not be implemented.
 
 | AWS Config Rule Name                                                                                                 | Summary                                                                                                 | Relevant Control IDs                 |
 | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| [dynamodb-autoscaling-enabled](https://docs.aws.amazon.com/config/latest/developerguide/dynamodb-autoscaling-enabled.html)                   | DynamoDB has autoscaling enabled.                              | CP-10, SC-5                                 |
+| [dynamodb-in-backup-plan](https://docs.aws.amazon.com/config/latest/developerguide/dynamodb-in-backup-plan.html)                   | DynamoDB tables are in an AWS Backup Plan.                              | CP-9(b), CP-10, SI-12                                 |
+| [dynamodb-table-encrypted-kms](https://docs.aws.amazon.com/config/latest/developerguide/dynamodb-table-encrypted-kms.html)                   | DynamoDB tables are encrypted in KMS.                              |  SC-13                                 |
```

Comment:
I think this rule needs a footnote. It's not implemented because it's unnecessary to implement

```
| [dynamodb-table-encrypted-kms](https://docs.aws.amazon.com/config/latest/developerguide/dynamodb-table-encrypted-kms.html)                   | DynamoDB tables are encrypted in KMS.<sup>[1](#dynamodb-table-encrypted-kms)</sup>                             |  SC-13                                 |
```

And then add a Footnotes section below
```
## Footnotes

<a name="dynamodb-table-encrypted-kms">1</a>: This rule is intentionally unimplemented. Server-side encryption at rest is enabled on all DynamoDB table data and [cannot be disabled](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/encryption.usagenotes.html#encryption.usagenotes.tabledata).
```